modules/nw-ptp-holdover-in-a-grandmaster-clock.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="holdover-in-a-grandmaster-clock_{context}"]
+= Holdover in a grandmaster clock with GNSS as the source
+
+Holdover allows the grandmaster (T-GM) clock to maintain synchronization performance when the GNSS source is unavailable. During this period, the T-GM clock relies on its internal oscillator and holdover parameters to reduce timing disruptions.
+
+You can define the holdover behavior by configuring the following holdover parameters in the `PTPConfig` custom resource (CR):
+
+`MaxInSpecOffset`:: Specifies the maximum allowed offset in nanoseconds. If the T-GM clock exceeds the `MaxInSpecOffset` value, it transitions to the `FREERUN` state (`gm.ClockClass` 248). 
+`LocalHoldoverTimeout`:: Specifies the maximum duration, in seconds, for which the T-GM clock remains in the holdover state before transitioning to the `FREERUN` state.
+`LocalMaxHoldoverOffSet`:: Specifies the maximum offset that the T-GM clock can reach during the holdover state in nanoseconds.
+
+If the `MaxInSpecOffset` value is less than the `LocalMaxHoldoverOffset` value, and the T-GM clock exceeds the maximum offset value, the T-GM clock transitions from the holdover state to the `FREERUN` state.
+
+[IMPORTANT]
+====
+Currently, even when the `LocalMaxHoldoverOffSet` value is less than the `MaxInSpecOffset` value, the holdover timeout occurs before the clock reaches the maximum offset. Therefore, it is recomended to set the `MaxInSpecOffset` field and the `LocalMaxHoldoverOffset` field to the same value, as the `LocalMaxHoldoverOffset` has no practical effect in this release. 
```

Comment:
The "even when" is throwing me off as the preceeding paragraph has the parameters the other way around: `MaxInSpecOffset` value is less than the `LocalMaxHoldoverOffset` value

How about: 

> If the `LocalMaxHoldoverOffSet` value is less than the `MaxInSpecOffset` value, the holdover timeout occurs before the clock reaches the maximum offset. To resolve this issue, set the `MaxInSpecOffset` field and the `LocalMaxHoldoverOffset` field to the same value. 

"because the value set for `LocalMaxHoldoverOffset` has no practical effect in this release." seems wrong given how setting a lower value does cause an issue. Best to drop the clause, IMO.